doc/api/events.md

```diff
@@ -155,8 +155,8 @@ myEmitter.emit('error', new Error('whoops!'));
   // Prints: whoops! there was an error
 ```
 
-As a best practice, developers should always register listeners for the
-`'error'` event:
+As a best practice, listeners should be registered for the `'error'` events
+always.
```

Comment:
That sounds alright, although I personally prefer s/registered/added/. I do not feel that strongly about it though.